Apps/backend/src/main.js

```diff
@@ -1,25 +1,31 @@
 import dgram from "node:dgram";
+import WebSocket, {WebSocketServer} from 'ws';
 
 const server = dgram.createSocket("udp4");
+const socket = new WebSocketServer({
+  port: 9000,
```

Comment:
Maybe it's better to have the WebSocket use ENV vars for host and port like the UDP socket.